pyscf/adc/__init__.py

```diff
@@ -24,25 +24,28 @@
 '''
 
 from pyscf import scf
+from pyscf import lib
 from pyscf.adc import uadc
 from pyscf.adc import radc
 
 def ADC(mf, frozen=None, mo_coeff=None, mo_occ=None):
     if not (frozen is None or frozen == 0):
         raise NotImplementedError
 
-    if isinstance(mf, scf.rhf.RHF):
-        return RADC(mf, frozen, mo_coeff, mo_occ)
-    elif isinstance(mf, scf.uhf.UHF):
-        return UADC(mf, frozen, mo_coeff, mo_occ)
-    elif isinstance(mf, scf.rhf.ROHF):
-        lib.logger.warn(mf, 'RADC method does not support ROHF reference. ROHF object '
-                        'is converted to UHF object and UADC method is called.')
-        mf = scf.addons.convert_to_uhf(mf)
+    if isinstance(mf, scf.uhf.UHF):
         return UADC(mf, frozen, mo_coeff, mo_occ)
+    #elif isinstance(mf, scf.rohf.ROHF):
+    #    lib.logger.warn(mf, 'RADC method does not support ROHF reference. ROHF object '
+    #                    'is converted to UHF object and UADC method is called.')
+    #    mf = scf.addons.convert_to_uhf(mf)
+    #    return UADC(mf, frozen, mo_coeff, mo_occ)
+    # TODO add ROHF functionality
+    elif isinstance(mf, scf.rhf.RHF):
+        return RADC(mf, frozen, mo_coeff, mo_occ)
```

Comment:
It seems to me these changes are spurious. You've merely changed the order of RHF and UHF and disabled ROHF altogether. I suggest resetting the file to the original version.

Reply:
Had to comment out the ROHF functionality due to some erroneous results. Need to investigate further with convert_to_uhf , which we plan to do later on.